frontend/packages/ux-editor/src/components/Elements/SubForm/SubFormUtils.test.ts

```diff
@@ -0,0 +1,25 @@
+import { SubFormUtils } from './SubFormUtils';
+import type { LayoutSet } from 'app-shared/types/api/LayoutSetsResponse';
+
+describe('SubFormUtils', () => {
+  const layoutSets: Array<LayoutSet> = [
```

Comment:
Also very nitpicking, but should we add another describe block for the actual utils-function, `findSubFormById`?
Also fair to wait with until there are actually added more functions